kernel/kernel-api/src/main/java/io/delta/kernel/internal/DeltaHistoryManager.java

```diff
@@ -162,6 +167,39 @@ public static long getEarliestRecreatableCommit(Engine engine, Path logPath)
         }
     }
 
+    /** Get the persisted commit info (if available) for the given delta file. */
+    public static Optional<CommitInfo> getCommitInfoOpt(
+            Engine engine,
+            Path logPath,
+            long version) throws IOException {
+        final FileStatus file = FileStatus.of(FileNames.deltaFile(logPath, version), 0, 0);
```

Comment:
FileStatus.of(FileNames.deltaFile(logPath, version),
 0, 0); <- please use named parameters here.